File: src/api/accessibility/simple/SimpleAccessibilityResponse.cs
using System;
using System.Collections.Generic;

namespace DVAN.API
{
    /// <summary>
    /// Multi-criteria response.
    /// </summary>
    public class SimpleAccessibilityResponse
    {
        /// <summary>
        /// Simple-accessibility values.
        /// </summary>
        public SimpleValue[] access { get; set; }
    }

    /// <summary>
    /// Simple value.
    /// </summary>
    public class SimpleValue
    {
        /// <summary>
        /// Range to closest facility.
        /// </summary>
        /// <example>123</example>
        public int first { get; set; }

        /// <summary>
        /// Range to second closest facility.
        /// </summary>
        /// <example>235</example>
        public int second { get; set; }

        /// <summary>
        /// Range to third closest facility.
        /// </summary>
        /// <example>412</example>
        public int third { get; set; }

        public SimpleValue(int first, int second, int third)
        {
            this.first = first;
            this.second = second;
            this.third = third;
        }
    }
}

File: src/main/java/org/tud/oas/api/accessibility/simple/SimpleAccessibilityResponse.java
package org.tud.oas.api.accessibility.simple;

import io.swagger.v3.oas.annotations.media.Schema;

@Schema(name = "SimpleAccessibilityResponse", description = """
        Simple Accessibility response.
        """)
public class SimpleAccessibilityResponse {
    @Schema(name = "access", description = """
            Simple-accessibility values.
            """)
    public SimpleValue[] access;

    public SimpleAccessibilityResponse(SimpleValue[] access) {
        this.access = access;
    }
}

@Schema(name = "SimpleValue", description = """
        Simple value.
        """)
class SimpleValue {
    @Schema(name = "first", description = """
            Range to closest facility.
            """, example = "123")
    public int first;

    @Schema(name = "second", description = """
            Range to second closest facility.
            """, example = "235")
    public int second;

    @Schema(name = "third", description = """
            Range to third closest facility.
            """, example = "412")
    public int third;

    public SimpleValue(int first, int second, int third) {
        this.first = first;
        this.second = second;
        this.third = third;
    }
}
